style.css

```diff
@@ -1,17 +1,51 @@
-nav {
-    position: fixed;
-    width: 100%;
-    top: 0;
-    left: 0;
-  }
-
-#navbar {
-   
-        display: inline;
-      }
-    
-      @media only screen and (max-width: 600px) {
-      body {
-        background-color: lightblue;
-      }
-    }
+/* All settings */
+* {
+  margin: 0;
+  padding: 0;
+  background-color: rgb(227, 230, 233);
+}
+
+/* Nav settings */
+.nav-container {
+  background-color: antiquewhite;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 25px;
+  gap: 25px;
+}
+
+/* First section */
+
+.first-section-container {
+
+  background-color: aqua;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 2rem;
+  padding: 1rem;
+  font-size: 23px;
+}
+
+/* Second section and proj
+ects */
```

Comment:
you almost got it. 👏👏👏
[ ] I kindly suggest you to improve the spaces in those lines. So you can pass easly throw the linters!